app/customer.py

```diff
@@ -0,0 +1,47 @@
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self,
+                 name: str,
+                 product_cart: dict,
+                 location: list[int],
```

Comment:
The type hint for `location` should be `tuple[int, int]` instead of `list[int]` to better represent a fixed pair of coordinates.